tests/test_event.py

```diff
@@ -1,7 +1,9 @@
 from datetime import datetime, timedelta
+import os
 import json
 import pytest
```

Comment:
Order imports

Reply:
fixed for next commit (something is seriously wrong with my auto-import!!!)